keanu-project/src/main/java/io/improbable/keanu/algorithms/variational/NonGradientOptimizer.java

```diff
@@ -12,8 +12,7 @@
 import org.apache.commons.math3.optim.nonlinear.scalar.ObjectiveFunction;
 import org.apache.commons.math3.optim.nonlinear.scalar.noderiv.BOBYQAOptimizer;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
```

Comment:
FYI - I've started documenting some IntelliJ settings at https://improbableio.atlassian.net/wiki/spaces/~616259109/pages/283574476/IntelliJ+settings 

Could you please enable the ones relating to import so that our commits are consistent on import order?